app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, dependent: :destroy
+  has_many :comments, dependent: :destroy
+  has_many :likes, dependent: :destroy
```

Comment:
- [ ] it would be better if you change `user` to `author`. as in the diagram in order to match the requirements you can do that like the below example: 
```ruby
has_many :(table name), foreign_key: 'author_id'
```

Reply:
@michael-duke the table name should be users but the foreign keys on the association tables should author

Reply:
Changes requested have been made ✅